src/lib/remark-plugins/rewrite-static-tutorials-assets/index.ts

```diff
@@ -70,7 +83,17 @@ export const rewriteStaticAssetsPlugin: Plugin = () => {
 				params.set('version', branchName)
 				params.set('asset', assetPath)
 			} else {
-				//  Otherwise, pass the unchanged path to a custom asset server for local dev
+				/**
+				 * Otherwise, pass the unchanged path to a custom asset server for tutorials repo local dev.
+				 *
+				 * @TODO Fix local tutorials images for dev-portal
+				 * https://app.asana.com/0/1202097197789424/1204908882543128
```

Comment:
🎫 [heres the ticket](https://app.asana.com/0/1202097197789424/1204908882543128) for that, will be a follow up